projects/packages/forms/src/blocks/contact-form/components/jetpack-field-datepicker.js

```diff
@@ -49,14 +49,40 @@ const JetpackDatePicker = props => {
 		} ),
 		style: blockStyle,
 	} );
+	const classNamesArray = attributes?.className ? attributes.className.split( ' ' ) : [];
+	const isNativeBrowserStyle = classNamesArray.includes( 'is-style-browser-native' );
+	const suffix = ! isNativeBrowserStyle
+		? `(${ DATE_FORMATS.find( f => f.value === dateFormat )?.label })`
+		: '';
+	const dateFormatInput = ! isNativeBrowserStyle ? (
```

Comment:
Wondering if this could be moved to prop directly? Just reads me nicer when code is located closer to where it'll end up.

I would try keep this inlined and do something like:

```js
extraFieldSettings={ ...( ! isNativeBrowserStyle ? [ /*element */ ] : [] ) }
```

Reply:
This was updates as well.